packages/client/foreman_ygg_migration/foreman_ygg_migration.spec

```diff
@@ -0,0 +1,34 @@
+Name: foreman_ygg_migration
+Version: 0.0.1
+Summary: A helper package to ease transition from yggdrasil 0.2.z to 0.4.z
+Release: 1%{?dist}
+License: MIT
+
+%if 0%{?rhel} >= 8
+Supplements: yggdrasil >= 0.4.0
+%endif
+Conflicts:   yggdrasil  < 0.4.0
+
+%description
+A helper package to ease transition from yggdrasil 0.2.z to 0.4.z
+
+%post
+if systemctl is-enabled yggdrasild >/dev/null 2>/dev/null; then
+	grep -Pq '^server'    /etc/yggdrasil/config.toml || sed -i 's/broker.*=/server =/' /etc/yggdrasil/config.toml
+	grep -Pq '^prefix'    /etc/yggdrasil/config.toml || echo 'prefix = "yggdrasil"' >>/etc/yggdrasil/config.toml
```

Comment:
This doesn't look idempotent. Perhaps `grep` should be stricter to check if the prefix isn't correct? Same below for `data-host`

Reply:
I was operating under the assumption that the configuration was generated by foreman. That would mean that the `data-host` and `prefix` either aren't there at all or are generated correctly (after https://github.com/theforeman/foreman/pull/10340 gets in). I could tighten it down, but I'm not sure if that's worth it